.github/workflows/screeners.yml

```
@@ -0,0 +1,217 @@
+name: Screeners workflow
+
+on: pull_request
+
+env:
+        SKIPCOMPONENTGOVERNANCEDETECTION: true
+
+        ConnectionString: ${{ secrets.AZURE_STORAGE_CONNECTION_STRING}}
+
+        BROWSERSLIST_IGNORE_OLD_DATA: true
```

```suggestion
        # Prevents failures on CI when "caniuse-lite" becomes outdated
        BROWSERSLIST_IGNORE_OLD_DATA: true
```

In general let's keep comments that describe the purpose of these variables